.github/workflows/validate-conventional-commits.yml

```diff
@@ -0,0 +1,12 @@
+name: Validate Conventional Commit Title
+on:
+  pull_request:
+    types: [opened, edited, reopened]
+
+jobs:
+  pr-title-linter:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: amannn/action-semantic-pull-request@v5.4.0
```

Comment:
Can we update this to use the same version that mobile is using `amannn/action-semantic-pull-request@c3cd5d1ea3580753008872425915e343e351ab54`

"As a general rule we don't use tags for actions, as a precaution against the action author's account being compromised. Tags can be moved, hashes cannot."

Reply:
Done